uitests/objectid/source/class/objectid/Application.js

```diff
@@ -0,0 +1,89 @@
+/* ************************************************************************
+
+   Copyright:
+
+
+   License:
+
+   Authors:
+
+ ************************************************************************ */
+
+/**
+ * This is the main application class of your custom application "dragndrop"
+ * 
+ * @asset(qx/icon/Tango/16/status/dialog-error.png)
+ */
+qx.Class.define("objectid.Application", {
+  extend: qx.application.Standalone,
+
+  members: {
+    main: function() {
+      this.base(arguments);
+
+      if (qx.core.Environment.get("qx.debug")) {
+        qx.log.appender.Native;
+        qx.log.appender.Console;
+      }
+
+      var root = this.getRoot();
+      
+      root.add(new qx.ui.basic.Label("Use the DOM Elements inspector to look at the widgets, " +
+      		"check out the 'data-object-id' attributes.<br>\n" +
+      		"<br>\n" +
+      		"Click the 'Run Test' button and then try this code in console:")
+        .set({ rich: true }), { left: 100, top: 20 });
+      root.add(new qx.ui.form.TextArea().set({ 
+        value: "var edtName = $(\"input[data-object-id='application/exampleEditor/edtName']\");\n" +
```

Comment:
Is this some ES6 syntax? What does this `$()` phrase do?

Reply:
this is jquery syntax

Reply:
it's jquery - it just an example to illustrate how something other than Qooxdoo can navigate the objects inside a Qooxdoo application (when unit testing this would be Selenium doing the query, it's just that jQuery is easier to explain for testing)